### PR TITLE
Add documentation for group add/remove

### PR DESCRIPTION
**Pull request by ryansch**

### CODE REVIEW

File: endpoints/groups.md [hunk 1]
```diff
@@ -89,3 +91,37 @@ On success, this returns ```204 No Content```. If the request is invalid, a ```4
...
+
+* ```POST /groups/:id/contacts.json``` adds the contacts specified by the `ids` param to the specified group.
+
+```json
+{
+  "ids": [1, 2, 3]
+}
+```
+
+On success, this returns ```204 No Content```. If the request is invalid, a ```422 Unprocessable Entity``` is returned with the error(s) in the response body. (See [422 Unprocessable Entity](https://github.com/outstand/api-docs/blob/master/422.md))
```
Comment (morganick) [inline endpoints/groups.md]: If one of the contacts can't be found, do we report that or do we still say `204`?

File: endpoints/groups.md [hunk 2]
```diff
@@ -89,3 +91,37 @@ On success, this returns ```204 No Content```. If the request is invalid, a ```4
...
+
+* ```DELETE /groups/:id/contacts.json``` removes the contacts specified by the `ids` param from the specified group.
+
+```json
+{
+  "ids": [1, 2, 3]
+}
+```
+
+On success, this returns ```204 No Content```. If the request is invalid, a ```422 Unprocessable Entity``` is returned with the error(s) in the response body. (See [422 Unprocessable Entity](https://github.com/outstand/api-docs/blob/master/422.md))
```
Comment (morganick) [inline endpoints/groups.md]: What would constitute an invalid request?

If we couldn't find the contact, we would consider that work completed and return a `204` right?

File: endpoints/groups.md (hunk 1 above)
Comment (ryansch) [reply endpoints/groups.md]: We just ignore it and add any others that are valid.

File: endpoints/groups.md (hunk 2 above)
Comment (ryansch) [reply endpoints/groups.md]: That's correct.  Right now you'd be hard pressed to get a 422 out of this endpoint.  This is mostly armor against future changes.

**Pull request merged**